content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,100 @@
 title: "Official Providers"
 weight: 20
 ---
-Upbound creates, maintains and supports a set of Crossplane providers called *official providers*.  
+Upbound creates, maintains and supports a set of Crossplane providers called **official providers**.  
 
-## Find official providers
-Identify official providers in the marketplace with the `by Upbound` gold seal.
+Upbound builds, tests and supports official providers. The Upbound official providers are open source under the Apache 2.0 license. Upbound recommends the official providers for all deployments.
```

Comment:
"Upbound creates, maintains and supports"
"Upbound builds, tests and supports official providers"

Feels duplicative?